app/src/test/java/org/simple/clinic/summary/linkId/LinkIdWithPatientViewControllerTest.kt

```diff
@@ -33,20 +34,27 @@ class LinkIdWithPatientViewControllerTest {
 
   private val uiEvents = PublishSubject.create<UiEvent>()
 
-  private val patientUuid = UUID.randomUUID()
+  private val patientUuid = UUID.fromString("755bfa1a-afa5-4c80-9ec7-57d81dff2ca1")
 
   private val identifier = Identifier(
-      value = patientUuid.toString(),
+      value = "40269f4d-f177-44a5-9db7-3cb8a7a53b33",
       type = Identifier.IdentifierType.random()
   )
 
+  private val identifierUuid = UUID.fromString("097a39e5-945f-44de-8293-f75960c0a54e")
```

Comment:
They are not the same thing.

- `identifierUuid` is the UUID of the generated record.
- `Identifier#value` is a string value that represents the unique ID. For a BP Passport, it will be a UUID string. For a drivers licence, it will be the license number.

Reply:
Oh yeah, I think I misunderstood the implementation. My bad 😅

Reply:
No problem, it can get confusing at times.